### ISSUE TITLE
Add Map to Event Series screens with city locations. 

### ISSUE THREAD

**Issue by tasomaniac** [gdg-x/frisbee#278]
This MapFragment can go into both `MainActivity` and `TaggedEventSeriesActivity`.

This will be much like the Google's websites about the event series like DevFests, IO Extended and Study Jams. 
It will have pins of cities that are organizing the event.

**Comment by friedger** [comment]
There is an menu to navigate or locate the event on maps using intents, so the map fragment should work together.

**Comment by tasomaniac** [comment]
Definitely.

**Comment by mauimauer** [comment]
Hmmmm, not sure if we would be served better by just intent jumping to Google Maps (unless we provide all the extra contextual functions GMap provides (routing etc.))

**Comment by tasomaniac** [comment]
I was thinking to have a map just like the one here with basic pins. 
https://www.womentechmakers.com/

**Comment by mauimauer** [comment]
I agree that stuff like that looks fancy, altough I am not sure of the direct benefit this provides for the user (especially on mobile)

**Comment by tasomaniac** [comment]
I think there is a potential benefit because it is really hard to find
events on the list. There are more than 100+ ioextended events around the
world. Although the list is sorted with the location, I find that the list
is complicated sometimes.

**Comment by mauimauer** [comment]
Not trying to stop you here, altough I'm still not convinced a map would really solve this problem. I'd prefer if we try to come up with a way to improve the list and leverage the sensor information we have. Maps are great for a lot of things (ie. navigating to a local event) but I still think this isn't one of those things.

**Comment by tasomaniac** [comment]
Design changes that address the problem I've described in the list may be more useful. Within a quick thought for example I can find improvements. For example, we can divide the list into sections that are happening near the user. Or we can divide them in country bases. 

That would be more useful than a map. But feature is a feature. I've already postponed the milestone of this issue. We can further postpone it.
